nodejs/week1/avg.js

```diff
@@ -0,0 +1,11 @@
+const args = process.argv.slice(2);
+const numbers = args
+  .map((arg) => parseFloat(arg))
```

Comment:
If you want to, you can simplify this block, by passing reference of this function, like `.map(parseFloat)`

Reply:
Thanks, simplified